src/components/Pages/ManageWiki/Features/EntityTable.vue

```diff
@@ -0,0 +1,246 @@
+        <v-dialog v-model="dialogDelete" max-width="500px">
+          <v-card>
+            <v-card-title class="headline">Are you sure you want to delete this mapping?</v-card-title>
+            <v-card-actions>
+              <v-spacer></v-spacer>
+              <v-btn color="blue darken-1" text @click="closeDelete">Cancel</v-btn>
+              <v-btn color="blue darken-1" text @click="deleteItemConfirm">OK</v-btn>
+              <v-spacer></v-spacer>
+            </v-card-actions>
+          </v-card>
+        </v-dialog>
+      </v-toolbar>
+    </template>
+    <template v-slot:[`item.actions`]="{ item }">
+      <v-icon
+          small
+        class="mr-2"
+        @click="editItem(item)"
+      >
+        edit
+      </v-icon>
+
+      <v-icon
+          small
+        @click="deleteItem(item)"
+      >
+        delete
+      </v-icon>
+    </template>
+  </v-data-table>
+
+    <v-dialog
+        v-model="dialog"
+        max-width="500px"
+    >
+        <template v-slot:activator="{ on, attrs }">
+            <v-btn
+                color="primary"
+                dark
+                class="space"
+                v-bind="attrs"
+                v-on="on"
+            >
+                Add New Mapping
+            </v-btn>
+        </template>
+        <v-card>
+        <v-card-title>
+            <span class="headline">{{ formTitle }}</span>
+        </v-card-title>
+
+        <v-card-text>
+          <v-form ref="mapping">
+            <v-container>
+            <v-row>
+                <v-col
+                cols="12"
+                sm="6"
+                md="4"
+                >
+                <v-text-field
+                    v-model="editedItem.local"
+                    label="My Wikibase (Your Instance)"
+                    :rules="[isValidEntityId]"
+                    validate-on-blur
+                ></v-text-field>
+                </v-col>
+                <v-col
+                cols="12"
+                sm="6"
+                md="4"
+                >
+                <v-text-field
+                    v-model="editedItem.wikidata"
+                    label="Wikidata"
+                    :rules="[isValidEntityId]"
+                    validate-on-blur
+                ></v-text-field>
+                </v-col>
+            </v-row>
+            </v-container>
+          </v-form>
+        </v-card-text>
+
+        <v-card-actions>
+            <v-spacer></v-spacer>
+            <v-btn
+            color="blue darken-1"
+            text
+            @click="close"
+            >
+            Cancel
+            </v-btn>
+            <v-btn
+            color="blue darken-1"
+            text
+            @click="save"
+            >
+            Save
+            </v-btn>
+        </v-card-actions>
+        </v-card>
+    </v-dialog>
+</div>
+</template>
+
+<script>
+export default {
+  name: 'EntityTable',
+  props: ['mappingName', 'entityValidation', 'entityName', 'storeAction'],
```

Comment:
I think passing a store action name to a component is a code smell.
The component should either be aware of the store or not. And here we have it work with the store but we don't tell it exactly in what context so it's a bit strange. 
Ultimately, this component would then not use the store and just communicate to a parent via events.